src/components/Comment.jsx

```diff
@@ -0,0 +1,16 @@
+import "./Comment.css";
+
+const Comment = ({ comment }) => {
+  return (
+    <div className="comment-container" key={comment.comment_id}>
```

Comment:
Is there something more semantic than a div you could use to mark up each comment? Currently you have an UL of divs.